<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/assets/assets/icon.png</normaloff>:/assets/assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>380</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_beacon"/>
    <addaction name="actionSave_beacon"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_path"/>
    <addaction name="actionSave_Path"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_beacon">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/assets/assets/open.png</normaloff>:/assets/assets/open.png</iconset>
   </property>
   <property name="text">
    <string>Open .beacon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_beacon">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/assets/assets/save.png</normaloff>:/assets/assets/save.png</iconset>
   </property>
   <property name="text">
    <string>Save .beacon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_path">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/assets/assets/open.png</normaloff>:/assets/assets/open.png</iconset>
   </property>
   <property name="text">
    <string>Open path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_Path">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/assets/assets/save.png</normaloff>:/assets/assets/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
